package.json

```diff
@@ -63,6 +63,7 @@
     "bignumber.js": "9.1.1",
     "date-fns": "2.30.0",
     "i18next": "22.5.1",
+    "lodash.uniq": "^4.5.0",
```

Comment:
Shouldn't we use lodash-es? Other than that, do we need 3rd party lib here? Can't we just `[...new Set(layers)]` instead of uniq ?

Reply:
I don't know which is the preferred library, and I don't have a strong preference either way. OK, dropping the 3rd party...